### ISSUE TITLE
Harbor SSL: missing key/cert causes log flooding

### ISSUE THREAD

**Issue by brenc** [savonet/liquidsoap#2590]
**Describe the bug**
If you have SSL configured like this:
```
settings.harbor.ssl.private_key.set('/app/blah.key')
settings.harbor.ssl.certificate.set('/app/blah.cert')
```
And if either the cert or the key doesn't exist and a request comes in, the logs are completely flooded with:

`Failed to accept new client: SSL: Certificate error: error:80000002:system library::No such file or directory`

**Expected behavior**
Report a failure once then fail.

**Version details**
 - OS: Docker
 - Version: v2.1.1

**Comment by brenc** [comment]
Also, when the cert or key ends up being an empty file, this floods the logs:

`2022/08/30 19:06:12 [harbor:2] Failed to accept new client: SSL: Certificate error: error:0480006C:PEM routines::no start line`

**Comment by vitoyucepi** [comment]
@brenc you're expecting a behavior like nginx?
Maybe it'll be a good idea to throw exception instead?

For example to reload certificates
```ruby
def reload_ssl(_)
	try
		settings.harbor.ssl.private_key.set('key.pem')
		settings.harbor.ssl.certificate.set('cert.pem')
		"ok"
	catch _ do
		"fail"
	end
end

server.register(
	namespace = "ssl",
	usage = "reload",
	description = "reload ssl",
	"reload",
	reload_ssl
)
```

**Comment by toots** [comment]
Thanks for reporting. This should be fixed in `main` and `rolling-release-v2.1.x`
